code/__defines/species.dm

```diff
@@ -99,5 +99,19 @@ SPECIES_NECROMORPH_LURKER,\
 SPECIES_NECROMORPH_UBERMORPH
 
 
+#define SPECIES_ALL_NECROMORPHS_NO_SLASHERS SPECIES_NECROMORPH,\
```

Comment:
this is unnecessary, species restrictions are handled at runtime in necro outfits